docs/release_notes/v0.10.0.md

```diff
@@ -0,0 +1,471 @@
+
+```bash
+helm del --purge dapr
+```
+
+If you previously installed Dapr using __Helm 3.X__:
+
+```bash
+helm uninstall dapr -n dapr-system
+```
+
+Update the Dapr repo:
+
+```bash
+helm repo update
+```
+
+If you installed Dapr with Helm to a namespace other than `dapr-system`, modify the uninstall command above to account for that.
+
+You can now follow [these](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#using-helm-advanced) instructions on how to install Dapr using __Helm 3__.
+
+Alternatively, you can use the CLI:
+
+```
+dapr init --kubernetes
+```
+
+#### Post installation
+
+Verify the control plane pods are running and are healthy:
+
+```
+dapr status -k
+
+dapr-sentry            dapr-system  True     Running  0.10.0    1d   2020-08-18 10:15.21  
+dapr-operator          dapr-system  True     Running  0.10.0    1d   2020-08-18 10:15.21  
+dapr-sidecar-injector  dapr-system  True     Running  0.10.0    1d   2020-08-18 10:15.21  
+dapr-placement         dapr-system  True     Running  0.10.0    1d   2020-08-18 10:15.21 
+```
+
+After Dapr 0.10.0 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
+This can be done with:
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
+
+### Multi Pub/Sub
+
+With 0.10.0, you can publish and subscribe to topics arriving from multiple pub/sub components.
+To this end, several changes have been made to both the publisher and consumer sides.
+
+#### Publisher
+
+An additional parameter has been added to tell Dapr which pub/sub component you're targetting.
+
+Old API:
+
+```
+POST http://localhost:<daprPort>/v1.0/publish/<topic>
+```
+
+New API:
+
+```
+POST http://localhost:<daprPort>/v1.0/publish/<pubsubname>/<topic>
+```
+
+#### Subscriber
+
+*Note: The following example in in Javascript but applicable to all languages*
+
+An additional `pubsubName` field has been added to the subscription payload to tell Dapr which pub/sub component this subscription is for.
+
+Old subscription:
+
+```javascript
+app.get('/dapr/subscribe', (req, res) => {
+    res.json([
+        {
+            topic: "newOrder",
+            route: "orders",
+        }
+    ]);
+})
+```
+
+#### CLI
+
+The `publish` command on the Dapr CLI now takes a required param to specifiy the pubsub component:
+
+<pre>
+dapr publish <b>--pubsub pubsub</b> --topic myevent --data '{ "name": "yoda" }'
+<pre>
+
+New subscription:
+
+```javascript
+app.get('/dapr/subscribe', (req, res) => {
+    res.json([
+        {
+            topic: "newOrder",
+            route: "orders",
+            pubsubName: "messagebus"
+        }
+    ]);
+})
+```
+
+### State retries
+
+The retry options for state operations have been removed. If you've never used retries, this is not a breaking change.
+The updated API references can be found [here](https://github.com/dapr/docs/blob/master/reference/api/state_api.md).
+
+### Java SDK
```

Comment:
Subscribe callback now should return response object: https://github.com/dapr/java-sdk/issues/310

Reply:
I'll add a general note for the different SDKs instead of mentioning one by one.

Reply:
Added